### PR TITLE
docs(core): refresh the example execution manifest for 0.8.0

### PR DESCRIPTION
**Pull request by gmboquet**
## Worklist item

**Item:** F10.5 (Retire misleading showcase examples -- acceptance criterion: "example manifest contains no ambiguous 'looks runnable' entries" / "ensure every README-linked example has a release execution status"). Documentation-accuracy correction found during a documentation-accuracy audit; also refreshes evidence gathered against the retired 0.7.0 wheel.

## Summary

`docs/example-execution-manifest.rst` had two concrete problems, both confirmed by
actually running things against current `release/0.8.0` (not inferred):

1. **Stale wheel evidence.** The "Release Execution Status" section's pass/fail record was
   gathered against the built `mixle-0.7.0` wheel (2026-07-09). Rebuilt the wheel from current
   source (`mixle-0.8.0.dev0`), installed it into a fresh bare venv (no extras, no editable
   install, no `PYTHONPATH`, Python 3.12 -- matching the doc's own stated methodology), and
   re-ran all 23 previously-recorded base-install examples plus the two new real-data flagships.
2. **Missing inventory rows.** `examples/flagship_heterogeneous_adult.py` and
   `examples/flagship_temporal_sunspots.py` (added 2026-07-11, PRs #319/#320, worklist F10.1/F10.2)
   were never added to the Inventory table -- the post-merge drift-fix commit
   (805855da, 2026-07-11) caught their missing `Classification:` tags and two other stale
   manifests (`release-checklists/0.8.0-repro-bundle.json`, `serialization_schema_manifest.json`,
   both regenerated via their own scripts) but missed this one, which has no generator script (it's
   a hand-maintained execution ledger by design -- pass/fail/blocked is evidence from actually
   running things, not something a script can infer).

## Fresh verification (this PR)

* Confirmed the two flagship examples are **not** stale -- both pass against current 0.8.0
  source with real data:
  * `flagship_temporal_sunspots.py`: mixle held-out log-likelihood/obs -2.0762 vs. hmmlearn
    -2.0776 (independent-baseline agreement).
  * `flagship_heterogeneous_adult.py`: mean log-density train -10.719, held-out -10.839
    (generalized, not memorized).
* Re-ran the 23 base-install examples: **19 passed**, the same **2 reclassified-to-blocked**
  (`skeptic_challenge_example.py` needs scikit-learn, `win_demo_example.py` needs torch) as
  before, and **2 exceeded the 90s budget** (`hierarchical_mixture_example.py`,
  `lookback_hmm_example.py`) under this run's extreme host contention (`uptime` load averages
  ~140-175 against 10 cores the entire pass). Re-ran each individually with a 15-minute allowance
  to separate "slow under load" from "hung": `lookback_hmm_example.py` completed cleanly in 86s
  (essentially at the 90s line -- it calls `optimize(..., max_its=1000, delta=None)`, which
  disables early-stopping and always runs the full 1000 iterations by design, so it was already
  close to budget); `hierarchical_mixture_example.py` did **not** complete even with 15 minutes
  (10x budget) while confirmed still actively computing throughout (steadily increasing CPU time,
  never crashed). That one's fit path was rewritten 4 days before this pass by the EM-monotonicity
  fix in #435 (2026-07-13) -- plausibly a real convergence-behavior change, not just host noise.
  Recorded both honestly as `timed_out` per the manifest's own evidence standard, with this
  distinction and a recommendation to re-verify on an unloaded runner, rather than silently
  dropping either from the "passed" list or asserting a regression I can't fully confirm on this
  host. Flagging `hierarchical_mixture_example.py` specifically as a separate follow-up task.
* Total example count corrected 55 -> 57 (both new flagships are top-level, not in a subdirectory).

## Public API impact

- [x] This PR does **not** change any public `__all__`.

## Claims impact

- [x] A claim is affected and the relevant docs / claim-evidence ledger are updated with evidence at the required grade (this *is* the claim-evidence ledger for examples/).

## Evidence

```
$ python -m build --wheel --outdir dist/ .                 -> Successfully built mixle-0.8.0.dev0-py3-none-any.whl
$ python3.12 -m venv bare-venv && pip install <wheel>       -> mixle 0.8.0.dev0, no extras
$ python -c "import mixle; print(mixle.__file__)"          -> resolves inside the bare venv's site-packages (non-editable, confirmed)
$ python examples/<each of 23 base-install examples>.py     -> 19 passed, 2 blocked (sklearn/torch missing, expected), 2 exceeded 90s (see doc note)
$ pip install datasets hmmlearn (into the same bare venv)
$ python examples/flagship_temporal_sunspots.py             -> passed, mixle -2.0762 vs hmmlearn -2.0776
$ python examples/flagship_heterogeneous_adult.py           -> passed, train -10.719 / held-out -10.839
```

## Checklist

- [x] Tests added/updated and passing; `ruff check` + `ruff format --check` clean (no Python changed).
- [x] Consumer suites for any edited module pass (no test reads this doc's content; `mixle/tests/example_classification_test.py` is unaffected -- it checks example docstrings, not this manifest).
- [x] I am not merging this PR myself, and I have not enabled auto-merge.

### CODE REVIEW

**Comment by gmboquet** [comment]
Companion doc-accuracy fix from the same audit pass: #526 (test-tiers.rst).
